package.json

```diff
@@ -66,8 +66,10 @@
     "@aws-sdk/client-sfn": "^3.358.0",
     "@aws-sdk/credential-providers": "^3.358.0",
     "@aws-sdk/property-provider": "^3.357.0",
+    "@google-cloud/logging-min": "^10.4.0",
```

Comment:
This is an experimental library, likely to be modified.

> Note: logging-min is experimental, and its feature surface is subject to change.

Do we really want to rely on an experimental package?

Reply:
Good point, we can use `@google-cloud/logging` instead